cloudinit/sources/DataSourceRbxCloud.py

```diff
@@ -197,6 +198,7 @@ def read_user_data_callback(mount_dir):
 
 
 class DataSourceRbxCloud(sources.DataSource):
+    dsname = "RbxCloud"
```

Comment:
Fix bug https://bugs.launchpad.net/cloud-init/+bug/1855196 by add dsname.

Reply:
Should I mark my commit in special way to reference in launchpad?

Reply:
We'll want `LP: #1855196` in the commit message that ends up in-tree.